nemo_aligner/utils/train_utils.py

```diff
@@ -38,7 +38,7 @@ def set_sync_funcs(ptl_model, forward_only):
         param_sync_func = ptl_model.sync_overlap_parameters
 
     # pipeline schedules will get these from ptl_model.model.config
-    for module in ptl_model.get_gpt_module_list():
+    for module in ptl_model.get_model_module_list():
```

Comment:
@gshennvm should I roll this back? I think that's required in the latest container, but not in the current version that's in main?

Reply:
Should indeed not make it into main